Assets/_nathan/scripts/MusicTracks.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace _nathan.scripts
+{
+[CreateAssetMenu(fileName = "NewMusicTrackList", menuName = "inkjam_SOs")]
+public class MusicTracks : ScriptableObject
```

Comment:
What is the purpose of this? Is it adding value over the existing approach of naming the clip in the Ink files and dropping it into `Resources/Music`?

Reply:
removed from current pr